vNext/src/OrleansRuntime/OrleansRuntime.csproj

```diff
@@ -21,26 +25,22 @@
         ..\..\..\src\OrleansRuntime\obj\**\*.cs;
         ..\..\..\src\OrleansRuntime\bin\**\*.cs" />
   </ItemGroup>
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Sdk">
-      <Version>1.0.0-alpha-20161104-2</Version>
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="NETStandard.Library">
-      <Version>1.6.1</Version>
-    </PackageReference>
+    <PackageReference Include="NETStandard.Library" Version="1.6.1" />
   </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\Orleans\Orleans.csproj" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+
   <!-- Begin Orleans: Without these lines the project won't build properly -->
   <!-- Set path to ClientGenerator.exe -->
   <PropertyGroup>
     <!-- Visual Studio or MsBuild .sln build -->
     <OrleansReferencesBase>$(ProjectDir)..\Bootstrap\</OrleansReferencesBase>
     <!-- TFS build -->
-    <OrleansReferencesBase Condition="'$(builduri)' != ''">$(TargetDir)..\net462\</OrleansReferencesBase>
+    <OrleansReferencesBase Condition="'$(builduri)' != ''">$(OutputPath)\Bootstrap\</OrleansReferencesBase>
```

Comment:
This is why we've PR text :-) "(No TargetDir available because of unneeded imports)."

Commons and CSharp targets file included by build and at this point $(TargetDir) is not available.

In the future we should have a clean SDK file which is packed into the Nuget package, and a different one which is for building Orleans. Not specifying/carrying the Bootstrap dir from the beginning is a "mistake".